addon/styles/addon.scss

```diff
@@ -70,7 +70,8 @@
 }
 
 .frost-list .container {
-	border-right: 1px solid $frost-color-lgrey-1;
+  height: calc(100vh - 2px);
```

Comment:
I meant to put calc(100% - 2px) here. 2px is the top and bottom border. Consumer should have it's calculated height (based on nav/info/footer) to let the contain expand.